kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/schema/builder.kt

```diff
@@ -208,10 +212,14 @@ fun JsonSchema.Builder.obj(dsl: JsonSchema.JsonObjectBuilder.() -> Unit = {}) =
 
 /**
  * Defines a [JsonSchema.JsonArray] node where the elements are of the type provided by [typeBuilder].
+iI * The length of the array can be specified using the [minItems] and [maxItems] keywords.
+ *
+ * @param minItems - minimum array length, default value is 0
+ * @param maxItems - maximum array length, default value is [Int.MAX_VALUE]
```

Comment:
```suggestion
 * The length of the array can be specified using the [minItems] and [maxItems] keywords.
 *
 * [minItems] - minimum array length, default value is 0
 * [maxItems] - maximum array length, default value is [Int.MAX_VALUE]
```

Reply:
I think the `@param` does nothing for KDoc (please correct me if I'm wrong) :)

Reply:
To be honest I've seen @param usage in `spring-security` project. It's also described here https://kotlinlang.org/docs/kotlin-doc.html#param-name